test/functional/timeout.py

```diff
@@ -11,6 +11,7 @@
 #  express or implied. See the License for the specific language governing 
 #  permissions and limitations under the License.
 
+import logging
 # Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License"). You
```

Comment:
double copyright notice? also the import should go below (both?) the copyright notice(s?)